ansible_collections/arista/avd/plugins/plugin_utils/schema/avdtodocumentationschemaconverter.py

```diff
@@ -228,6 +228,7 @@ def type(self, schema: dict):
             "bool": "Boolean",
             "dict": "Dictionary",
             "list": "List",
+            "password": "Password",
```

Comment:
Now I am in doubt. When reading the doc, I think password is not clear for the user. So do we really want this to be a different type at all? Or should it be a "format" of type "str" ?

Reply:
Then format could be `bgp_password` to save one of the keys. And then we just insert the validation as a format checker.

Reply:
So I moved the PR to draft for now given that we need to reach consensus regarding how to implement this - but leveraging format can probably be a good option as well - this version is what we initially discussed to implement but now it is only a matter of moving things around :) that probably will go in 4.0!